Jenkinsfile

```diff
@@ -1,19 +1,39 @@
 pipeline {
     agent none
+    options {
+        buildDiscarder(logRotator(daysToKeepStr: '30'))
+    }
+    triggers {
+        GenericTrigger(causeString: 'Travis builds succeeded', token: '5331146c7f6f99b995e8')
+    }
     stages {
         stage('Build') {
             parallel {
 
+                stage('macOS 10.13/Apple clang-902.0.39.2/llvm-6.0.1 (Debug/ASAN)') {
+                    agent { label 'macos' }
+                    environment {
+                        PATH="/usr/local/opt/llvm/bin:$PATH"
```

Comment:
Does it work if we don't have this PATH line? Our FIndClangTools cmake stuff actually expect that not to be the case (it's written to seek out those folders to get clang-tidy and clang-format) and in fact adding that to PATH might mean we end up building with the clang version in there, rather than the stock Xcode AppleClang version that we expect.